src/components/StatisticsPage.js

```diff
@@ -3,41 +3,43 @@ import { inject, observer } from "mobx-react"
 import pattern from '../assets/images/pattern.svg'
 import { BridgeStatistics } from './index'
 import { TransactionsStatistics } from './TransactionsStatistics'
+import { BRIDGE_MODES } from '../stores/utils/bridgeMode'
 
 @inject("RootStore")
 @observer
 export class StatisticsPage extends React.Component {
 
   render(){
-    const { homeStore, foreignStore } = this.props.RootStore
+    const { homeStore, foreignStore, bridgeMode } = this.props.RootStore
+    const isNativeToErc = bridgeMode === BRIDGE_MODES.NATIVE_TO_ERC
     return(
       <div className="statistics-page">
         <div className='statistics-left-container' />
         <div className='statistics-page-container'>
           <div className='statistics-bridge-container'>
             <span className='statistics-bridge-title statistics-title'>Bridge Statistics</span>
               <BridgeStatistics
-                gasValue={454600}
                 users={homeStore.statistics.finished ? homeStore.statistics.users.size : ''}
                 totalBridged={homeStore.statistics.finished ? homeStore.statistics.totalBridged.toString() : ''}
                 homeBalance={homeStore.balance}
                 homeSymbol={homeStore.symbol}
+                homeNativeSupplyTitle={isNativeToErc}
                 foreignSymbol={foreignStore.symbol}
                 foreignSupply={foreignStore.totalSupply} />
           </div>
           <div className='statistics-transaction-container'>
             <div className='statistics-deposit-container'>
-              <span className='statistics-deposit-title statistics-title'>Network Deposits</span>
+              <span className='statistics-deposit-title statistics-title'>Tokens Deposits</span>
```

Comment:
The logic when we use `erc-to-erc` and `erc-to-native` modes are reversed.
* `native-to-erc`: tokens are deposited by sending coins from Home and withdrawn by sending tokens from Foreign
* `erc-to-erc`: tokens are deposited by sending tokens from Foreign and  withdrawn by sending tokens from Home
* `erc-to-native`: tokens are deposited by sending tokens from Foreign and  withdrawn by sending coins from Home

The current layout of lables "Tokens Deposits" and "Tokens Withdrawals" confuses. 
![image](https://user-images.githubusercontent.com/20793260/46806994-a94a0880-cd71-11e8-9c03-476da45e2762.png)
Since the amount of withdrawn tokens cannot be more than the amount of deposited tokens.

Reply:
Fixed